src/main/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelEventGenerator.java

```diff
@@ -0,0 +1,57 @@
+/*
+   Copyright 2019 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.gerrit.plugins.eiffel.events.generators;
+
+import java.net.InetAddress;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.UnknownHostException;
+
+public class EiffelEventGenerator {
+    protected static final String META_SOURCE_NAME = "Eiffel Gerrit Plugin";
+
+    private static final int DEFAULT_SSH_PORT = 29418;
+    private static final String DEFAULT_GERRIT_BASE_URL = "ssh://gerritmirror:" + DEFAULT_SSH_PORT;
```

Comment:
Isn't gerritmirror some Ericsson internal server hostname or is it something default for gerrit service.
Should be default to gerrit service, if there is any default. Should not have hardcoded Ericsson internal stuff in the code.

Reply:
We've removed this fallback value.

Reply:
ok.